src/jloptions.c

```diff
@@ -66,6 +67,7 @@ static const char opts[]  =
     " -H, --home <dir>          Set location of `julia` executable\n"
     " --startup-file={yes|no}   Load ~/.juliarc.jl\n"
     " --handle-signals={yes|no} Enable or disable Julia's default signal handlers\n\n"
+    " -s, --page-size={N}       Set the GC page size\n\n"
```

Comment:
This probably shouldn't have a short name. It might also be nicer to group these somehow. Something like `--config.gc.region.size={N}` ?